core/Scheduler/Timetable.php

```diff
@@ -19,12 +19,15 @@
 class Timetable
 {
     const TIMETABLE_OPTION_STRING = "TaskScheduler.timetable";
+    const RETRY_OPTION_STRING = "TaskScheduler.retryList";
 
     private $timetable;
+    private $retryList;
 
     public function __construct()
     {
         $this->readFromOption();
+        $this->readRetryList();
```

Comment:
maybe that one is actually not needed to call if we only read it when needed in the get/save